docs/conf.py

```diff
@@ -6,6 +6,7 @@
 from recommonmark.parser import CommonMarkParser
 
 sys.path.insert(0, os.path.abspath('..'))
+sys.path.append(os.path.dirname(__file__))
```

Comment:
The first insert is the root path, ie - `docs/../`, for our code, the second insert is the `docs/` path.